swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -63,6 +63,14 @@ func fsModifyPrivacy(setting: PrivacySetting, of view: UIView) {
     FS.addClass(view, className: setting.rawValue)
 }
 
+func fsModifyPrivacy(setting: PrivacySetting, views: [UIView?]) {
```

Comment:
`fsModifyPrivacy(setting: PrivacySetting, views: [UIView?])` is method overloading here to accommodate `views` as a convenience feature, correct? So you have the ability to pass in a list of views to add privacy rules on each.

A suggestion is that rather than overload `fsModifyPrivacy`, you convert the original function into a variadic one.  For example,
```
func fsModifyPrivacy(setting: PrivacySetting, of views: UIView...) {
    views.forEach { view in
        for view in views {
            FS.addClass(view, className: setting.rawValue)
        }
    }
}
```

What threw me off a bit at first was I saw lots of code using `fsModifyPrivacy(setting: setting, of: view)` and didn't grok how that related back to `fsModifyPrivacy(setting: PrivacySetting, views: [UIView?])`.  Then I saw that there was a pre-existing `fsModifyPrivacy(setting: PrivacySetting, of view: UIView)` and things made more sense.  I feel like rather than having two functions that do more or less the same thing, you might have one variadic, which then has a consistent call signature.  Thoughts?

Reply:
Sounds like a good idea so that there is just one method. Keeping it simple.